app/plan.go

```diff
@@ -51,7 +53,32 @@ func (s *planService) FindByName(name string) (*appTypes.Plan, error) {
 
 // DefaultPlan implements DefaultPlan method of PlanService interface
 func (s *planService) DefaultPlan() (*appTypes.Plan, error) {
-	return s.storage.FindDefault()
+	plan, err := s.storage.FindDefault()
+	if err != nil && err == appTypes.ErrPlanDefaultNotFound {
+		plans, err := s.storage.FindAll()
+		if err != nil {
+			log.Errorf("Error listing Plans: %s", err)
+			return nil, appTypes.ErrPlanDefaultNotFound
+		}
+		if len(plans) == 0 {
+			configMemory, _ := config.GetInt("docker:memory")
```

Comment:
Just a detail, I would extract the default plan creation to a separate private function, to let the code easier do read.